packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -161,3 +177,24 @@
         }
     }
 }
+
+.toggle-name {
```

Comment:
Can this be scoped to avoid collisions?

Reply:
I don't entirely know what you meant but I've changed the class name to `.p2p-toggle-name`?